geometry_msgs/Vector3 pos
geometry_msgs/Vector3 lin_vel
RPY rpy

